sources/packages/backend/apps/load-test-gateway/src/auth/guards/jwt-auth.guard.ts

```diff
@@ -0,0 +1,29 @@
+import { ExecutionContext, Injectable } from "@nestjs/common";
+import { Reflector } from "@nestjs/core";
+import { AuthGuard } from "@nestjs/passport";
+import { IS_PUBLIC } from "../../utils/auth.constants";
+
+/**
+ * Enable global validation for authenticated users and
+ * also allow the use of the "Public" decorator to skip
+ * the authentication validation when needed.
+ */
+@Injectable()
+export class JwtAuthGuard extends AuthGuard("jwt") {
```

Comment:
Is there a current need for this guard or anything planned for the future?
If not I would recommend removing it.

Reply:
Without this guard, PassportStrategy isn't performing any interception/validation. Am I missing something here?

Reply:
PassportStrategy is validating
- Token signature using public key
- Token expiration
- Audience
The AZP is used in SIMS-Api because of the multiple providers and the need to validate them individually. Is it the same case here?

Reply:
My bad, the jwt Guard is needed.